Source/PersonalityInsightsV3/PersonalityInsights.swift

```diff
@@ -0,0 +1,274 @@
+/**
+ * Copyright IBM Corporation 2016
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+import RestKit
+
+/**
+ The Watson Personality Insights service uses linguistic analytics to extract a spectrum
+ of cognitive and social characteristics from the text data that a person generates
+ through blogs, tweets, forum posts, and more.
+ */
+public class PersonalityInsights {
+    
+    /// The base URL to use when contacting the service.
+    public var serviceURL = "https://gateway.watsonplatform.net/personality-insights/api"
+    
+    /// The default HTTP headers for all requests to the service.
+    public var defaultHeaders = [String: String]()
+    
+    private let credentials: Credentials
+    private let domain = "com.ibm.watson.developer-cloud.PersonalityInsightsV3"
+
+    /**
+     Create a `PersonalityInsights` object.
+     
+     - parameter username: The username used to authenticate with the service.
+     - parameter password: The password used to authenticate with the service.
+     */
+    public init(username: String, password: String) {
+        credentials = Credentials.basicAuthentication(username: username, password: password)
+    }
```

Comment:
I think it makes more sense to specify the `version` parameter here in the `init` instead of each class method.

That will also help maintain consistency throughout the SDK since other services accept the `version` parameter in the `init` (for example, see [Conversation.swift](https://github.com/watson-developer-cloud/ios-sdk/blob/master/Source/ConversationV1/Conversation.swift#L48)).